src/containers.js

```diff
@@ -70,42 +70,35 @@ async function handle(url, tabId) {
   }
 
   const hostIdentity = identities.find((identity) => identity.cookieStoreId === hostMap.cookieStoreId);
-  const tabIdentity = identities.find((identity) => identity.cookieStoreId === currentTab.cookieStoreId);
+  let targetCookieStoreId;
 
   if (!hostIdentity) {
     if (preferences.defaultContainer) {
       const defaultContainer = await buildDefaultContainer(
           filterByKey(preferences, prefKey => prefKey.startsWith('defaultContainer')),
           url
       );
-      const defaultCookieStoreId = defaultContainer.cookieStoreId;
-      const defaultIsNoContainer = defaultCookieStoreId === NO_CONTAINER.cookieStoreId;
-      const tabHasContainer = currentTab.cookieStoreId !== NO_CONTAINER.cookieStoreId;
-      const tabInDifferentContainer = currentTab.cookieStoreId !== defaultCookieStoreId;
-      const openInNoContainer = defaultIsNoContainer && tabHasContainer;
-      if ((tabInDifferentContainer && !openInNoContainer) || openInNoContainer) {
-        console.debug('Opening', url, 'in default container', defaultCookieStoreId, defaultContainer.name);
-        return createTab(
-            url,
-            currentTab.index + 1, currentTab.id,
-            currentTab.openerTabId,
-            defaultCookieStoreId);
-      }
+      targetCookieStoreId = defaultContainer.cookieStoreId;
+      console.debug('Going to open', url, 'in default container', targetCookieStoreId, defaultContainer.name);
```

Comment:
Not sure about having logs in the release. Don't want to populate someone's console with our logs.

Reply:
I commented out the in to be able to comment it in once #96 is implemented